web/src/PickNeighbourhoodMode.svelte

```diff
@@ -262,62 +286,20 @@
   </div>
 
   <div slot="map">
-    <GeoJSON data={neighbourhoods} generateId>
+    <GeoJSON data={neighbourhoods} promoteId="name">
```

Comment:
Honestly `JoinedData` could use better docs or more examples. We need to first tell maplibre to use the `name` property as the unique ID, so the feature selector for `setFeatureState` works. Then this all works out.

Reply:
I did figure out `JoinedData.idCol`, but I missed the `GeoJSON.promoteId` - thanks for figuring it out!